tensorflow_datasets/image/mini_imagenet.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow_datasets.public_api as tfds
+import os
+import tensorflow as tf
+import pickle
+
+
+_MINI_IMAGENET_DESCRIPTION = """\
+miniImageNet (Vinyals et al., 2016) is a modified version of the ILSVRC-12
+dataset (Russakovsky et al., 2015), in which 600 images for each of 100 classes
+were randomly chosen to be part of the dataset. We rely on the class split used
+by Ravi & Larochelle (2017). These splits use 64 classes for training, 16 for
+validation, and 20 for test. All images are of size 84 × 84 pixels.
+"""
+
+_MINI_IMAGENET_URL_HOME = \
+    "https://github.com/renmengye/few-shot-ssl-public#miniimagenet"
+_MINI_IMAGENET_URL_DOWNLOAD = \
+    "https://drive.google.com/uc?export=download&id=16V_ZlkW4SsnNDtnGmaBRq2OoPmUOc5mY"
+
+_MINI_IMAGENET_CITATION = """\
+@inproceedings{vinyals2016matching,
+  title={Matching networks for one shot learning},
+  author={Vinyals, Oriol and Blundell, Charles and Lillicrap, Timothy and Wierstra, Daan and others},
+  booktitle={Advances in neural information processing systems},
+  pages={3630--3638},
+  year={2016}
+}
+"""
+
+_NUM_CLASSES = 100
+_NUM_CLASSES_TRAIN = 64
+_NUM_CLASSES_VALIDATION = 16
+_NUM_CLASSES_TEST = 20
+
+
+class MiniImagenet(tfds.core.GeneratorBasedBuilder):
+  """mini_imagenet: a modified version of the ILSVRC-12 dataset."""
+
+  VERSION = tfds.core.Version("1.0.2")
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+      builder=self,
+      description=_MINI_IMAGENET_DESCRIPTION,
+      features=tfds.features.FeaturesDict({
+        "image": tfds.features.Image(),
```

Comment:
Sorry, here will be 4 spaces of indentation. Similarly other places too. Please check some image datasets.
For eg. [this](https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/image/cycle_gan.py).

Reply:
@ChanchalKumarMaji Thanks for the suggestion. I have changed indentation at line breaks.